siriuspy/siriuspy/clientarch/devices.py

```diff
@@ -6,243 +6,19 @@
-        'SI-08M1:DI-BPM',
-        'SI-08M2:DI-BPM',
-        'SI-08C1:DI-BPM-1',
-        'SI-08C1:DI-BPM-2',
-        'SI-08C2:DI-BPM',
-        'SI-08C3:DI-BPM-1',
-        'SI-08C3:DI-BPM-2',
-        'SI-08C4:DI-BPM',
-        'SI-09M1:DI-BPM',
-        'SI-09M2:DI-BPM',
-        'SI-09C1:DI-BPM-1',
-        'SI-09C1:DI-BPM-2',
-        'SI-09C2:DI-BPM',
-        'SI-09C3:DI-BPM-1',
-        'SI-09C3:DI-BPM-2',
-        'SI-09C4:DI-BPM',
-        'SI-10M1:DI-BPM',
-        'SI-10M2:DI-BPM',
-        'SI-10C1:DI-BPM-1',
-        'SI-10C1:DI-BPM-2',
-        'SI-10C2:DI-BPM',
-        'SI-10C3:DI-BPM-1',
-        'SI-10C3:DI-BPM-2',
-        'SI-10C4:DI-BPM',
-        'SI-11M1:DI-BPM',
-        'SI-11M2:DI-BPM',
-        'SI-11C1:DI-BPM-1',
-        'SI-11C1:DI-BPM-2',
-        'SI-11C2:DI-BPM',
-        'SI-11C3:DI-BPM-1',
-        'SI-11C3:DI-BPM-2',
-        'SI-11C4:DI-BPM',
-        'SI-12M1:DI-BPM',
-        'SI-12M2:DI-BPM',
-        'SI-12C1:DI-BPM-1',
-        'SI-12C1:DI-BPM-2',
-        'SI-12C2:DI-BPM',
-        'SI-12C3:DI-BPM-1',
-        'SI-12C3:DI-BPM-2',
-        'SI-12C4:DI-BPM',
-        'SI-13M1:DI-BPM',
-        'SI-13M2:DI-BPM',
-        'SI-13C1:DI-BPM-1',
-        'SI-13C1:DI-BPM-2',
-        'SI-13C2:DI-BPM',
-        'SI-13C3:DI-BPM-1',
-        'SI-13C3:DI-BPM-2',
-        'SI-13C4:DI-BPM',
-        'SI-14M1:DI-BPM',
-        'SI-14M2:DI-BPM',
-        'SI-14C1:DI-BPM-1',
-        'SI-14C1:DI-BPM-2',
-        'SI-14C2:DI-BPM',
-        'SI-14C3:DI-BPM-1',
-        'SI-14C3:DI-BPM-2',
-        'SI-14C4:DI-BPM',
-        'SI-15M1:DI-BPM',
-        'SI-15M2:DI-BPM',
-        'SI-15C1:DI-BPM-1',
-        'SI-15C1:DI-BPM-2',
-        'SI-15C2:DI-BPM',
-        'SI-15C3:DI-BPM-1',
-        'SI-15C3:DI-BPM-2',
-        'SI-15C4:DI-BPM',
-        'SI-16M1:DI-BPM',
-        'SI-16M2:DI-BPM',
-        'SI-16C1:DI-BPM-1',
-        'SI-16C1:DI-BPM-2',
-        'SI-16C2:DI-BPM',
-        'SI-16C3:DI-BPM-1',
-        'SI-16C3:DI-BPM-2',
-        'SI-16C4:DI-BPM',
-        'SI-17M1:DI-BPM',
-        'SI-17M2:DI-BPM',
-        'SI-17C1:DI-BPM-1',
-        'SI-17C1:DI-BPM-2',
-        'SI-17C2:DI-BPM',
-        'SI-17C3:DI-BPM-1',
-        'SI-17C3:DI-BPM-2',
-        'SI-17C4:DI-BPM',
-        'SI-18M1:DI-BPM',
-        'SI-18M2:DI-BPM',
-        'SI-18C1:DI-BPM-1',
-        'SI-18C1:DI-BPM-2',
-        'SI-18C2:DI-BPM',
-        'SI-18C3:DI-BPM-1',
-        'SI-18C3:DI-BPM-2',
-        'SI-18C4:DI-BPM',
-        'SI-19M1:DI-BPM',
-        'SI-19M2:DI-BPM',
-        'SI-19C1:DI-BPM-1',
-        'SI-19C1:DI-BPM-2',
-        'SI-19C2:DI-BPM',
-        'SI-19C3:DI-BPM-1',
-        'SI-19C3:DI-BPM-2',
-        'SI-19C4:DI-BPM',
-        'SI-20M1:DI-BPM',
-        'SI-20M2:DI-BPM',
-        'SI-20C1:DI-BPM-1',
-        'SI-20C1:DI-BPM-2',
-        'SI-20C2:DI-BPM',
-        'SI-20C3:DI-BPM-1',
-        'SI-20C3:DI-BPM-2',
-        'SI-20C4:DI-BPM',
-        'SI-01M1:DI-BPM',
-        )
+    BPMS_TB = _BPMSearch.get_names({'sec': 'TB'})
```

Comment:
@xresende, there is an issue here. This commands will be executed in import time. Which means an error will be raised if there is no connection with the Constants server.

Reply:
This is indeed not acceptable. Thanks!